{ 'squat': { 'left_arm': [],
             'left_leg': [ { 'duration': 10.088,
                             'name': u'stand_up_5_left',
                             'positions': [ 0.0,
                                            -0.2557,
                                            -1.57,
                                            3.1,
                                            1.56,
                                            -0.2537],
                             'starttime': 0.0}],
             'neck': [],
             'right_arm': [],
             'right_leg': [ { 'duration': 10.088,
                              'name': u'stand_up_5_right',
                              'positions': [ -0.0,
                                             0.2556,
                                             1.57,
                                             -3.1,
                                             -1.57,
                                             0.2478],
                              'starttime': 0.0}],
             'torso': []}}
